src/main/scala/com/gu/elasticsearchmonitor/CloudwatchMetrics.scala

```diff
@@ -57,20 +58,20 @@ class CloudwatchMetrics(env: Env, cloudWatch: AmazonCloudWatch) {
       val sumTotalDiskSpace = dataNodes.map(_.dataTotal).sum
       val maxJvmHeapUsage = dataNodes.maxBy(_.jvmHeapUsedPercent).jvmHeapUsedPercent
       List(
-        metricDatum("MinAvailableDiskSpace", minAvailableDiskSpace.toDouble, StandardUnit.Bytes, defaultDimensions, now),
-        metricDatum("SumAvailableDiskSpace", sumAvailableDiskSpace.toDouble, StandardUnit.Bytes, defaultDimensions, now),
-        metricDatum("SumTotalDiskSpace", sumTotalDiskSpace.toDouble, StandardUnit.Bytes, defaultDimensions, now),
-        metricDatum("MaxJvmHeapUsage", maxJvmHeapUsage.toDouble, StandardUnit.Bytes, defaultDimensions, now))
+        metricDatum("MinAvailableDiskSpace", minAvailableDiskSpace.toDouble, StandardUnit.BYTES, defaultDimensions, now),
+        metricDatum("SumAvailableDiskSpace", sumAvailableDiskSpace.toDouble, StandardUnit.BYTES, defaultDimensions, now),
+        metricDatum("SumTotalDiskSpace", sumTotalDiskSpace.toDouble, StandardUnit.BYTES, defaultDimensions, now),
+        metricDatum("MaxJvmHeapUsage", maxJvmHeapUsage.toDouble, StandardUnit.PERCENT, defaultDimensions, now))
```

Comment:
The change of unit here is intentional as we noticed a bug: https://github.com/guardian/elastic-search-monitor/pull/252#discussion_r2410846893